README.md

```diff
@@ -147,3 +147,13 @@ To deploy CanDIGv2, follow one of the available install guides in `docs/`:
 * [Tox Deployment Guide](./docs/install-tox.md)
 
 View additional Makefile options with `make help`.
+
+
+## Services and Components
+
+The follwing table lists the details from the Data Flow Diagram in the "Overview" section.
+
+| Service/Component Name                  | Source            | Notes                  |
+| ----------------------                  |----------------   |--------                |
```

Comment:
Looks good, could you start to add the additional components and tag who needs to provide the info for it?

Reply:
Yea I guess I would need to start somewhere. Perhaps @zxenia to begin the Katsu stuff. Thanks!